game/assets/jak1/subtitle/game_subtitle_it.gd

```diff
@@ -0,0 +1,2605 @@
+("sksp0301" :hint #x608 (0 "DAXTER" "NON COLPIRE QUEGLI ESPLOSIVI!"))
+("sksp0302" :hint #x609 (0 "DAXTER" "STAGLI LONTANO E VEDRAI CHE CE LA FAREMO!"))
+("sksp0303" :hint #x60a (0 "DAXTER" "COLPISCI L'ECO BLU PER ACCELERARE!"))
+("sksp0304" :hint #x60b (0 "DAXTER" "EVITA GLI ALBERI, JAK! CI RALLENTANO!"))
+("sksp0305" :hint #x60c (0 "DAXTER" "PIÙ ECO BLU PRENDI, PIÙ ACCELERI!"))
+("sksp0306" :hint #x60d (0 "DAXTER" "FAI ATTENZIONE AGLI ESPLOSIVI!"))
+("sksp0307" :hint #x60e (0 "DAXTER" "STAI LONTANO DAI POZZI!"))
+("sksp0308" :hint #x60f (0 "DAXTER" "FORSE DOVREI GUIDARE IO!"))
+("sksp0309" :hint #x610 (0 "DAXTER" "C'È UN LURKER!"))
+("sksp0310" :hint #x611 (0 "DAXTER" "PRENDILI, JAK!"))
+("sksp0311" :hint #x612 (0 "DAXTER" "NE ABBIAMO PASSATO UNO!"))
+("sksp0312" :hint #x613 (0 "DAXTER" "AAAH, UNO DEI LURKER CI HA SUPERATO!"))
+("sksp0313" :hint #x614 (0 "DAXTER" "SONO MOLTO AVANTI!"))
+("sksp0314" :hint #x615 (0 "DAXTER" "YAA-AH! SIAMO PRIMI!"))
+("sksp0315" :hint #x616 (0 "DAXTER" "PIÙ VELOCE, JAK, PIÙ VELOCE!"))
+("sksp0316" :hint #x617 (0 "DAXTER" "SALTA, SALTAAA!"))
+("sksp0317" :hint #x618 (0 "DAXTER" "CONTINUA COSÌ!"))
+("sksp0318" :hint #x619 (0 "DAXTER" "YAAAAAAAAHH!"))
+("sksp0319" :hint #x61a (0 "DAXTER" "SARÀ UN FINALE COMBATTUTO!"))
+("sksp0320" :hint #x61b (0 "DAXTER" "ARRIVERANNO AL DETONATORE PRIMA DI NOI!"))
+("sksp0321" :hint #x61c (0 "DAXTER" "CI SIAMO RIUSCITI! ABBIAMO EVITATO CHE DISTRUGGESSERO IL PASSAGGIO!"))
+
+("asstvb25" :hint #x61d
+  (10 "KEIRA" "BEL LAVORO! LI AVETE PRECEDUTI AL DETONATORE!")
+  (184 "KEIRA" "IL LABORATORIO DEL SAGGIO ROSSO È PROPRIO DI FRONTE A VOI.")
+  (348 "KEIRA" "ANDATE AD ATTIVARE IL TELETRASPORTO, COSÌ POSSIAMO RAGGIUNGERVI.")
+  )
+
+;; -----------------
+;; village3
+;; -----------------
+
+("assistant-village3-reminder"
+  (19 "KEIRA" "RAGAZZI, CONTINUATE A RACCOGLIERE LE BATTERIE,")
+  (89 "KEIRA" "È IL SOLO MODO PER POTER PROSEGUIRE IL VIAGGIO VERSO NORD.")
+  (192)
+  )
+
+("minershort-introduction-gnawers"
+  (11 "GORDY" "EHI VOI DUE! PERCHÉ NON CERCATE DI RENDERVI UTILI?")
+  (100 "GORDY" "DEI LURKER STANNO SCAVANDO NELLA GROTTA OSCURA LÀ SOPRA.")
+  (200 :offscreen "GORDY" "SEMBRA STIANO CERCANDO I MANUFATTI DEI PRECURSOR...")
+  (279)
+  (286 :offscreen "GORDY" "SE LI POSSONO PRENDERE ANCHE TUTTI, PER QUEL CHE M'IMPORTA.")
+  (363)
+  (368 :offscreen "WILLARD" "GIÀ, PER QUEL CHE C'IMPORTA.")
+  (431)
+  (448 "GORDY" "WILLARD, DÀ DA MANGIARE AL CANARINO.")
+  (502)
+  (503 "GORDY" "A ME INTERESSANO SOLTANTO LE GEMME!")
+  (572 "GORDY" "MA NON RIESCO A PRENDERLE PERCHÉ QUANDO VENGONO ESTRATTE,")
+  (659 "GORDY" "PURTROPPO FANNO CROLLARE TUTTO!")
+  (715)
+  (717 "GORDY" "CERCA DI ELIMINARE I LURKER CHE ROSICCHIANO I TRONCHI DI SUPPORTO,")
+  (818 "GORDY" "COSÌ MI SARAI DI GRANDE AIUTO.")
+  (875)
+  (886 "GORDY" "E MUOVITI!")
+  )
+
+("minershort-introduction-orbs"
+  (16 "???" "GAH... EHI, GORDY! ACCIPICCHIA, PENSO CHE ABBIAMO DELLE VISITE!")
```

Comment:
Willard and Gordy's names can be known before this plays, obscuring is unnecessary.